web/src/inbox_ui.js

```diff
@@ -310,6 +312,14 @@ function format_topic(stream_id, topic, topic_unread_count, latest_msg_id) {
         is_collapsed: collapsed_containers.has(STREAM_HEADER_PREFIX + stream_id),
         mention_in_unread: unread.topic_has_any_unread_mentions(stream_id, topic),
         latest_msg_id,
+        // The following two fields are not specific to this context, but this is the
```

Comment:
Maybe clarify which two feels, in case this code is reordered or modified in the future.

Reply:
Done.